setsubun.p8

```diff
@@ -42,6 +47,13 @@ function _init()
 	)
 end
 
+function init_menu()
+	menuitem(1, "lang: "..lang, function()
+		lang=(lang=="ja") and "en" or "ja"
```

Comment:
This probably needs to use the new localized string functions to be useful?

Reply:
Yeah, I think I'm going to remove the menu option after all.